manifests/crc/default-db-included-installation/openshift/kruize_openshift.yaml

```diff
@@ -0,0 +1,589 @@
+apiVersion: v1
```

Comment:
Looks like you've mistakenly added this file. This isn't needed.

Reply:
Thanks, updated